### ISSUE TITLE
ABOUT Training scripts

### ISSUE THREAD

**Issue by popov1212** [ZhengYinan-AIR/Diffusion-Planner#17]
Thanks for your great work！

in the training scripts torch_run.sh

If I use the nuplan data set, how could I
Set the following path?
TRAIN_SET_PATH=?
TRAIN_SET_LIST_PATH=?

I try to use the original *.db file path : **/dataset/nuplan -v1.1/splits/trainval as TRAIN_SET_PATH please check the picture attached below 

And I set the TRAIN_SET_LIST_PATH=”**/normalization.json"
when I use nuplan dataset. I wonder if it's right?

![Image](https://github.com/user-attachments/assets/2c071718-b144-4d38-9608-9391b402149f)

Big Thanks ahead for your help!

**Comment by ZhengYinan-AIR** [comment]
Hello, these two paths are for using the preprocessed nuPlan data. The core [code](https://github.com/ZhengYinan-AIR/Diffusion-Planner/blob/main/diffusion_planner/data_process/data_processor.py) has already been released, and I will provide the running script this week.

**Comment by popov1212** [comment]
yes, I found the core code but once running the torch_run.sh many errors come up, Maybe cause I haven't preprocessed the nuplan data. 

Thanks ahead for your running script of nuplan data preprocessing script! 

BTW, is your coming running script only used for nuplan?

**Comment by yu-chang-sysu** [comment]
> yes, I found the core code but once running the torch_run.sh many errors come up, Maybe cause I haven't preprocessed the nuplan data.
> 
> Could you update the method of using the script in the README including how to set these two path after nuplan dataset preprossing script released:TRAIN_SET_PATH and TRAIN_SET_LIST_PATH?
> 
> Thanks ahead for your upcoming script for preprocessing Nuplan DataSet.
> 
> BTW, is your upcoming script only used for nuplan?
hello, I found a same problem of yours, do you solve this problem or write a useful script to preprocess the nuplan data?

**Comment by yu-chang-sysu** [comment]
> yes, I found the core code but once running the torch_run.sh many errors come up, Maybe cause I haven't preprocessed the nuplan data.
> 
> Could you update the method of using the script in the README including how to set these two path after nuplan dataset preprossing script released:TRAIN_SET_PATH and TRAIN_SET_LIST_PATH?
> 
> Thanks ahead for your upcoming script for preprocessing Nuplan DataSet.
> 
> BTW, is your upcoming script only used for nuplan?



hello, I found a same problem of yours, do you solve this problem or write a useful script to preprocess the nuplan data?

**Comment by popov1212** [comment]
The author has release the preprocessing script, pls checkout the updated commit

**Comment by yu-chang-sysu** [comment]
yes, i have seen it. Do you found any problem when running the train code? I writed the error I found in a new issue.

**Comment by popov1212** [comment]
Hi, there! Have you finished the data preprocess？
If your answer is yes, TRAIN_SET_LIST_PATH should be the path of nuplan_train.json I think.

**Comment by yu-chang-sysu** [comment]
There is a  diffusion_planner_training.json appear after preprocessing the data, and I use its path to be the TRAIN_SET_LIST_PATH and then something wrong happen. Thank u for your advice and I will try it tomorrow

**Comment by popov1212** [comment]
Thank you. Cause I am still working on the data preprocessing, pls keep me posted if there are any updates
